packages/client/ui/react-ui/src/components/embed/v3/EmbeddedCheckoutV3IFrame.tsx

```diff
@@ -0,0 +1,66 @@
+import { useCrossmint } from "@/hooks";
+import { useEffect, useRef, useState } from "react";
+
+import {
+    CrossmintEmbeddedCheckoutV3Props,
+    EmbeddedCheckoutV3IFrameEmitter,
+    embeddedCheckoutV3IncomingEvents,
+    embeddedCheckoutV3OutgoingEvents,
+} from "@crossmint/client-sdk-base";
+import { IFrameWindow } from "@crossmint/client-sdk-window";
+
+export function EmbeddedCheckoutV3IFrame(props: CrossmintEmbeddedCheckoutV3Props) {
+    const [iframeClient, setIframeClient] = useState<EmbeddedCheckoutV3IFrameEmitter | null>(null);
+    const [height, setHeight] = useState(0);
+
+    const { apiClient } = useCrossmint();
+    const ref = useRef<HTMLIFrameElement>(null);
+
+    async function createIFrameClient(iframe: HTMLIFrameElement) {
```

Comment:
check https://github.com/Crossmint/crossmint-sdk/pull/588/files - moved to common service